src/schema/objects/metadata.yaml

```diff
@@ -1113,6 +1170,53 @@ EthicsApprovals:
   type: array
   items:
     type: string
+EventIdentifier:
+  name: EventIdentifier
+  display_name: Event Identifier
+  description: |
+    The message sent to the eye-tracker which disambiguates the belonging
+    of the sample to a certain event, condition or group.
+    This can be a unique trial identifier, or in the case of
+    continuous presentation (for example video) the annotated event of interest.
+  type: string
+EyeCameraSettings:
+  name: EyeCameraSettings
+  display_name: Eye Camera Settings
+  description: |
+    A field to store any settings that influence the resolution
+    and quality of the eye imagery.
+    Autowhitebalance? Changes in sharpness?
+  type: object
+  items:
+    type: object
+EyeTrackerDistance:
+  name: EyeTrackerDistance
+  display_name: Eye-Tracker Distance
+  description: |
+    Distance (in meters) between the eye-tracker and the participant eye(s).
+  type: number
```

Comment:
Change this to `type: array` so as to accommodate a more holistic EyeTrackerDistance recording as per suggestions from [this paper](https://pubmed.ncbi.nlm.nih.gov/25953668/).

Suggested description:
`description: Distance (in meters) between the eye-tracker and the participant eye(s). Distance can either be expressed as a single numeric value indicating the Z-axis distance between the participant's eye and the eyetracker, or 3 dimensional tuple indicating the X, Y, Z distances between the participant's eye and the eyetracker.`